manifest/openshift4/coreos/ncp-openshift4.yaml

```diff
@@ -452,6 +457,57 @@ data:
 
 
 
+---
```

Comment:
This part should be removed because it's for ocp3. I have not change it in NCP upstream not to generate it so we have to manually delete it each time when syncing manifests.

Reply:
Thanks and fixed.